### DISCUSSION TITLE
problem with ojs cell

### DISCUSSION THREAD

**Discussion by Pierre9344** [quarto-dev/quarto-cli#8844] [Q&A] [answered]
### Description

Hello,
I have some problem with ojs cell with the latest version of quarto.
I opened an issue #8840 but it was closed because my example was not valid. I did the corrections do my code but I still have issues. I indicated them in the bug report but I can't reopen it so I decided to create a post here to avoid to duplicated issue.

**Comment by mcanouil** [comment, marked as answer]
Please don't open alternate thread.
Issues/Discussions can be reopened (not necessarily by you).
Follow up on #8840.

See my reply:
- https://github.com/quarto-dev/quarto-cli/issues/8840#issuecomment-1959552270

Thank you!
